### PR TITLE
 Fix federated cloud share icons overlap in button

### LINKED ISSUE #30819: Federated cloud share icons overlap in button
See screenshot (22.2.3 here)

image

### PR DESCRIPTION
**Pull request by Lucarnosky**
Added margin right the the icon to avoid overlap over the write

Fix #30819

### CODE REVIEW

**Review by CarlSchwan — APPROVED** [review]

**Comment by CarlSchwan** [comment]
Thanks! This looks good to me :)

Please fix the DCO https://github.com/nextcloud/server/pull/31596/checks?check_run_id=5573597201 :)

**Comment by szaimen** [comment]
> Did you test the solution? Couldn't get it to work.

yes, that is also what I thought... margin right doesn't seem right...

File: apps/federatedfilesharing/css/settings-personal.scss [hunk 1]
```diff
@@ -30,10 +30,13 @@
 
 .social-diaspora {
 	@include icon-color('social-diaspora', 'federatedfilesharing', $color-black);
+	margin-right: 10px;
```
Comment (artonge) [inline apps/federatedfilesharing/css/settings-personal.scss:33]:
```suggestion
	padding-left: 26px;
```

**Comment by CarlSchwan** [comment]
Padding left is probably more likely to work

**Review by artonge — CHANGES_REQUESTED** [review]

**Comment by Lucarnosky** [comment]
Fixed

**Review by artonge — APPROVED** [review]

**Comment by artonge** [comment]
All good :)

@Lucarnosky can you fix DCO? https://github.com/nextcloud/server/pull/31596/checks?check_run_id=5578043641

**Comment by Lucarnosky** [comment]
@artonge 
Squashed and add self signed. Let me know if everything is ok now

**Comment by artonge** [comment]
> @artonge Squashed and add self signed. Let me know if everything is ok now

Signe-off is present, but I think it needs to be at the end of the commit message

**Comment by Lucarnosky** [comment]
> > @artonge Squashed and add self signed. Let me know if everything is ok now
> 
> Signed-off is present, but I think it needs to be at the end of the commit message

Sorry for the mess, fixed the commit

**Comment by artonge** [comment]
> Sorry for the mess, fixed the commit

Commit sha: [3ea53c3](https://github.com/nextcloud/server/pull/31596/commits/3ea53c304a77b4fb2c192b51d4461ba2921c8da4), Author: Lock Garnock, Committer: Lock Garnock; Expected "Lock Garnock [lucarnosky@gmail.com](mailto:lucarnosky@gmail.com)", but got "Lucarnosky [lucarnosky@gmail.com](mailto:lucarnosky@gmail.com)".

SHould be "Lock Garnock" instead of "Lucarnosky"

**Comment by Lucarnosky** [comment]
> > Sorry for the mess, fixed the commit
> 
> Commit sha: [3ea53c3](https://github.com/nextcloud/server/pull/31596/commits/3ea53c304a77b4fb2c192b51d4461ba2921c8da4), Author: Lock Garnock, Committer: Lock Garnock; Expected "Lock Garnock [lucarnosky@gmail.com](mailto:lucarnosky@gmail.com)", but got "Lucarnosky [lucarnosky@gmail.com](mailto:lucarnosky@gmail.com)".
> 
> SHould be "Lock Garnock" instead of "Lucarnosky"

DCO Passed :)

**Comment by artonge** [comment]
Thanks for this @Lucarnosky ! :)

**Comment by Lucarnosky** [comment]
Thank you guys for patience and amazing work. Glad to contribute

**Pull request merged**